src/app-exception-filters/main-exception.filter.ts

```diff
@@ -29,4 +29,12 @@ export class MainExceptionFilter implements ExceptionFilter {
                     : exception.message,
         });
     }
+
+    private buildFullExceptionMessage(exception: Error): string {
```

Comment:
jak ten log wyglada w koncowej wersji?

Reply:
i jaka jest roznica pomiedzy tym co bylo

Reply:
jest taka różnica, że poza tym, że mam customowy error AuthException to jeszcze wiem, dlaczego został rzucony bo jako cause dołączony jest error z np. Axiosa jeśli od spowodował AuthException. Tu jest najprościej to pokazane, 
oczywiście mogę załączyć screen z testów manualnych jak to wygląda dla tej apki.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause